multibody/parsing/parsing_doxygen.h

```diff
@@ -708,4 +711,19 @@ object. Units are kg⋅m² for revolute joints, and kg for prismatic joints.
 @see drake::multibody::JointActuator, @ref tag_drake_gear_ratio,
 @ref reflected_inertia "Reflected Inertia"
 
+@subsection tag_drake_screw_thread_pitch drake:screw_thread_pitch
+
+- SDFormat path: `//model/joint/drake:screw_thread_pitch`
```

Comment:
I'll file chaser PR to fix this to `//model/joint/screw_thread_pitch` given that it's part of SDFormat proper